src/test/java/org/minions/devfund/benjamin/MovieDatabaseTest.java

```diff
@@ -0,0 +1,118 @@
+package org.minions.devfund.benjamin;
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+/**
+ * Class to test MovieDatabase class.
+ */
+public class MovieDatabaseTest {
+    /**
+     * Test Add a new movie and actors.
+     */
+    @Test
+    public void testAddMovieNewMovieAndActors() {
+        MovieDatabase movieDatabase = new MovieDatabase();
+        String newMovie = "Batman Forever";
+        movieDatabase.addMovie(newMovie, new String[]{"Crhistian Bale", "Morgan Freeman"});
+        assertEquals(1, movieDatabase.getMovieList().size());
+        assertEquals(newMovie, movieDatabase.getMovieList().get(0).getName());
+        assertEquals(2, movieDatabase.getActorList().size());
+    }
+
+    /**
+     * Test add movie that already exist.
+     */
+    @Test
+    public void testAddMovieAlreadyExist() {
+        MovieDatabase movieDatabase = new MovieDatabase();
+        String newMovie = "Batman Forever";
+        movieDatabase.addMovie(newMovie, new String[]{"Crhistian Bale", "Morgan Freeman"});
+        movieDatabase.addMovie(newMovie, new String[]{"Crhistian Bale", "Morgan Freeman"});
+        assertEquals(1, movieDatabase.getMovieList().size());
+        assertEquals(newMovie, movieDatabase.getMovieList().get(0).getName());
+        assertEquals(2, movieDatabase.getActorList().size());
+    }
+
+    /**
+     * Test add rating.
+     */
+    @Test
+    public void testAddRatingNewMovie() {
+        MovieDatabase movieDatabase = new MovieDatabase();
+        String newMovie = "Batman Forever";
+        movieDatabase.addMovie(newMovie, new String[]{"Crhistian Bale", "Morgan Freeman"});
+        final double ten = 10.0;
+        movieDatabase.addRating(newMovie, ten);
+        assertEquals(ten, movieDatabase.getMovieList().get(0).getRating(), 0.0);
+    }
+
+    /**
+     * Test update rating.
+     */
+    @Test
+    public void testUpdateRating() {
+        MovieDatabase movieDatabase = new MovieDatabase();
+        String newMovie = "Batman Forever";
+        movieDatabase.addMovie(newMovie, new String[]{"Crhistian Bale", "Morgan Freeman"});
+        final double ten = 10.0;
+        final double fifteen = 15.0;
+        movieDatabase.addRating(newMovie, ten);
+        movieDatabase.updateRating(newMovie, fifteen);
+        assertEquals(fifteen, movieDatabase.getMovieList().get(0).getRating(), 0.0);
+    }
+
+    /**
+     * Test get best actor.
+     */
+    @Test
+    public void testGetBestActor() {
+        MovieDatabase movieDatabase = new MovieDatabase();
+        String newMovie = "Batman Forever";
+        String newMovie2 = "NewMovie2";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "NewMovie2" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)